g11n-ws/modules/md-data-api-bundleimpl/src/main/java/com/vmware/vip/messages/data/dao/impl/OneComponentDao.java

```diff
@@ -23,130 +23,129 @@
 import com.vmware.vip.common.constants.ConstantsUnicode;
 import com.vmware.vip.common.i18n.resourcefile.LocalJSONReader;
 import com.vmware.vip.common.i18n.resourcefile.ResourceFilePathGetter;
-import com.vmware.vip.common.i18n.resourcefile.ResourceFileWritter;
 import com.vmware.vip.messages.data.bundle.BundleConfig;
+import com.vmware.vip.messages.data.bundle.JSONUtil;
 import com.vmware.vip.messages.data.dao.api.IOneComponentDao;
 import com.vmware.vip.messages.data.dao.exception.DataException;
 import com.vmware.vip.messages.data.dao.model.ResultI18Message;
 import com.vmware.vip.messages.data.exception.BundleException;
 
 /**
- * This java class is used to handle translation bundle file or translation
- * cache for single component
+ * This java class is used to handle translation bundle file or translation cache for single
+ * component
  */
 @Repository
 public class OneComponentDao implements IOneComponentDao {
-	
-	private static Logger logger = LoggerFactory.getLogger(OneComponentDao.class);
-	
-	@Autowired
-	private BundleConfig bundleConfig;
-	
-	
-	
-	@Override
-	public ResultI18Message get(String productName, String version, String component, String locale) throws DataException{
```

Comment:
why need to modify this method?

Reply:
I have checked it, it need to sort the bundle file content when sync the translations from GRM